src/main/java/reposense/model/RepoConfiguration.java

```diff
@@ -96,7 +96,8 @@ public static void merge(List<RepoConfiguration> repoConfigs, List<AuthorConfigu
 
             if (matchingRepoConfig == null) {
                 logger.warning(String.format(
-                        "Repository %s is not found in repo-config.csv.", authorConfig.getLocation()));
+                        "Repository %s (branch %s) is not found in repo-config.csv.",
+                        authorConfig.getLocation(), authorConfig.getBranch()));
```

Comment:
Sorry, I just remembered that the branch is an optional field in author config. 
Can you tweak this message, such that it wont print the branch name if the field is using default branch?
This is to prepare the structure for https://github.com/reposense/RepoSense/issues/865

Reply:
Resolved